### PR TITLE
1959 button toggle and badge documentation

### PR DESCRIPTION
**Pull request by MCdanja**
Button toggle and badge documentation added.

### CODE REVIEW

File: source/includes/_documentation.md [hunk 1]
```diff
@@ -4853,6 +4853,185 @@ The angular Button:
...
+    @Test
+    public void basicUnderlineButtonToggleTest() {
+        String pressedAttribute = "aria-pressed";
+        basicButtonToggleUnderline.click();
+        basicButtonToggleUnderline.has().attr(pressedAttribute, "true");
+        basicButtonToggleUnderline.click();
+        basicButtonToggleUnderline.has().attr(pressedAttribute, "false");
+    }
+```
+
```
Comment (ViktarR) [inline source/includes/_documentation.md:4880]: image
Please move code example to related element.

File: source/includes/_documentation.md [hunk 2]
```diff
@@ -4853,6 +4853,185 @@ The angular Button:
...
+    }
+   } 
+```
+#### <a href="https://material.angular.io/components/badge/overview" target="_blank">Badge overview</a>
+
+Badge is located in the following class:
+
+- __Java__: _com.epam.jdi.light.angular.elements.common.Badge_
+
+![Badge](../images/angular/badge.png)
```
Comment (ViktarR) [inline source/includes/_documentation.md:5014]: Badge image is not shown.
image
The reason can be in naming, in file name PNG in upper case.

Comment (gbuh) [inline source/includes/_documentation.md:5014]: The image is not loaded. The reason is an incorrect extension. You download the badge.PNG file, but link it to badge.png.

Comment (MCdanja) [reply source/includes/_documentation.md]: Fixed

Comment (MCdanja) [reply source/includes/_documentation.md]: Fixed

**Pull request merged**